app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/FahrplanFragment.java

```diff
@@ -388,7 +389,7 @@ private void addRoomTitleViews(
     /**
      * jump to current time or lecture, if we are on today's lecture list
      */
-    private void scrollToCurrent(@NonNull List<Lecture> lectures, int day, int height) {
+    private void scrollToCurrent(int height) {
```

Comment:
Can you rename height to boxHeight? This way we use same name for same thing. An another approach would be to create a value type BoxHeight.